### ISSUE TITLE
Bug in auto import feature

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/TypeScript#19878]
_From @sseppola on November 9, 2017 9:54_

- VSCode Version: Code 1.18.0 (dcee2202709a4f223185514b9275aa4229841aa7, 2017-11-08T12:32:39.870Z)
- OS Version: Darwin x64 16.7.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
sort-lines|Tyr|1.3.0
Bookmarks|ale|0.17.0
unique-lines|bib|1.0.0
npm-intellisense|chr|1.3.0
path-intellisense|chr|1.4.2
vscode-eslint|dba|1.4.3
gitlens|eam|6.0.0
Go|luk|0.6.67
java|red|0.13.0
transpose|v4r|0.0.5
vscode-java-debug|vsc|0.2.0
vscode-java-pack|vsc|0.2.0

(1 theme extensions excluded)

---

Steps to Reproduce:

1. Restarted VSCode
2. Typed 'log' + enter in a javascript project to trigger a console.log snippet

This triggered VSCode to include the following import statement at the top of my file:
```js
import { log } from '../../../../../../../../../../../../../../Library/Caches/typescript/2.6/node_modules/@types/gulp-util';
```

Reproduces without extensions: Yes

Only reproducible once after launching the application.

_Copied from original issue: Microsoft/vscode#37907_

**Comment by mjbvz** [comment]
@sseppola There should be multiple entries for `log` in this case, including the old snippet one. Can you please check this

---

Moved this issue TS because we should never create auto-imports like the above in the first place

**Comment by sseppola** [comment]
@mjbvz I'm not entirely sure what you mean by entries here. 
If you mean the suggestions in the dropdown, then yes there are multiple, but I've configured Code to show my snippets first. I'm quite sure I didn't click the wrong suggestion.
If you mean entries as in text in my file then yes, a `log` was left where I was trying to insert `console.log`.

**Comment by mjbvz** [comment]
@sseppola Yes I meant the suggestions. Here's what I see:

<img width="520" alt="screen shot 2017-11-10 at 12 44 54 pm" src="https://user-images.githubusercontent.com/12821956/32677868-fd448b50-c614-11e7-9903-3c54dd57bfbd.png">

**Comment by sseppola** [comment]
When I restart Code and type log I see this:
<img width="523" alt="screen shot 2017-11-11 at 11 32 20" src="https://user-images.githubusercontent.com/4065840/32688591-335c8f2a-c6d4-11e7-9ef5-1abb1945e61d.png">

After inserting the above suggestion and trying again I get this:
<img width="531" alt="screen shot 2017-11-11 at 11 31 34" src="https://user-images.githubusercontent.com/4065840/32688597-41b739c6-c6d4-11e7-8fa4-d1868a723f53.png">
